app/shared/user-agent-client.js

```diff
@@ -17,10 +17,15 @@ import { logger } from './logging';
 import { UserAgentHttpClient } from './user-agent-http-client';
 
 class UserAgentClient extends EventEmitter {
-  constructor() {
+  // You'd think { initialConnectDelayMillis = 50, maxConnectDelayMillis = 10000 } would work.  In
+  // fact, you'd be crying in your beer, just like me: it doesn't when you call simply `new
+  // UserAgentClient()`.
+  constructor(options) {
```

Comment:
If you're referring to ` = {}` as a "style change", that assertion would be wrong. It's not a stylistic or opinionated syntax preference, it's the only correct js syntax.